erpnext/patches/v11_0/update_delivery_trip_status.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2017, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	frappe.reload_doc('stock', 'doctype', 'delivery_trip')
+	frappe.reload_doc('stock', 'doctype', 'delivery_stop', force=True)
+
+	for trip in frappe.get_all("Delivery Trip"):
+		trip_doc = frappe.get_doc("Delivery Trip", trip.name)
+

```

Comment:
this code is almost duplicate of `update_status` in `delivery_trip.py`

Reply:
Yeah, this is intentional, since I don't want to update the `modified` dates of the old trips, and have them pushed up in the list view.